### ISSUE TITLE
[template/i18n] reports error for interpolation

### ISSUE THREAD

**Issue by mattlewis92** [angular-eslint/angular-eslint#682]
**Description and reproduction of the issue**

```JSON
{
  "rules": {
        "@angular-eslint/template/i18n": [
          "error",
          {
            "checkId": true,
            "checkAttributes": false,
            "ignoreTags": [":svg:title", ":svg:desc", ":svg:tspan"]
          }
        ],
  }
}
```

Interpolation bindings get reported by the rule with `Each element containing text node should have an i18n attribute.` but this is no actual english text in there to extract. This was working ok with 12.3.x so I guess it was caused by https://github.com/angular-eslint/angular-eslint/pull/665
```HTML
<ng-template #errorMessage>
  {{ error.title }}
</ng-template>
```

**Versions**

| package                                   | version |
| ----------------------------------        | ------- |
| `@angular-eslint/eslint-plugin-template`  | `12.4.1` |
| `@angular-eslint/template-parser`         | `12.4.1` |
| `@typescript-eslint/parser`               | `4.28.5` |
| `ESLint`                                  | `7.29.0` |
| `node`                                    | `14.17.0` |

```sh
# Please run `npx ng version` in your project and paste the full output here:
Angular CLI: 12.2.6
Node: 14.17.0
Package Manager: npm 6.14.13
OS: darwin x64

Angular: 12.2.6
... animations, cli, common, compiler, compiler-cli, core
... elements, forms, language-service, localize
... platform-browser, platform-browser-dynamic, router
... service-worker

Package                         Version
---------------------------------------------------------
@angular-devkit/architect       0.1202.1
@angular-devkit/build-angular   12.2.6
@angular-devkit/core            12.1.4
@angular-devkit/schematics      12.2.6
@angular/cdk                    12.2.1
@angular/google-maps            12.2.1
@schematics/angular             12.2.6
ng-packagr                      12.2.1
rxjs                            7.3.0
typescript                      4.3.5
```

- [x] I have tried restarting my IDE and the issue persists.
- [x] I have updated to the latest supported version of the packages and checked my `ng version` output per the instructions given here.

**Comment by rafaelss95** [comment]
Sorry for that, @mattlewis92. Unfortunately, I got confused by the name `DEFAULT_BOUND_TEXT_ALLOWED_PATTERN` as I described at #683 and messed up this.

It should be solved by #683.

**Comment by mattlewis92** [comment]
> Sorry for that, @mattlewis92. Unfortunately, I got confused by the name `DEFAULT_BOUND_TEXT_ALLOWED_PATTERN` as I described at [#683 (comment)](https://github.com/angular-eslint/angular-eslint/pull/683#issue-999566599) and messed up this.
> 
> It should be solved by #683.

Thanks for the super quick turnaround on a fix, it's much appreciated!! 🙌

**Comment by DDeivid** [comment]
We have currently having the same issue. @JamesHenry any timeline or ETA when this fix will be released?

**Comment by JamesHenry** [comment]
@DDeivid It was available in a published version (as all successful `master` commits are) a few minutes after it was merged.

We outline this approach in the README - https://github.com/angular-eslint/angular-eslint#package-versions

You can install any of the canary releases that have been published since #683 was merged a couple of days ago

**Comment by DDeivid** [comment]
> @DDeivid It was available in a published version (as all successful `master` commits are) a few minutes after it was merged.
> 
> We outline this approach in the README - https://github.com/angular-eslint/angular-eslint#package-versions
> 
> You can install any of the canary releases that have been published since #683 was merged a couple of days ago

Thanks for this 🙏
